.editorconfig

```diff
@@ -7,7 +7,7 @@ root = true
 [*]
 end_of_line = lf
 insert_final_newline = true
-charset = utf8
+charset = utf-8
```

Comment:
Err, this is unrelated, but didn't seem worth its own PR. Let me know if y'all disagree and I'll turn it into a separate PR.